plugins/ptp_operator/metrics/ptp4lParse.go

```diff
@@ -52,10 +52,15 @@ func (p *PTPEventManager) ParsePTP4l(processName, configName, profileName, outpu
 			}
 		}
 	} else if strings.Contains(output, " port ") {
-		portID, role, syncState := extractPTP4lEventState(output)
```

Comment:
assign result of isFollowerOnly(ptp4lCfg) to a variable so that the function does not have to be rerun again on line 60.